WORKSPACE.bazel

```diff
@@ -0,0 +1,76 @@
+## workspace( name = "zexe" ) ## DO NOT ENABLE - clashes with local_repository below
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+################################################
+################  RUST SUPPORT  ################
+load("//bzl/external:rust_deps.bzl", "rust_bootstrap")
+rust_bootstrap()
+
+load("@io_bazel_rules_rust//rust:repositories.bzl",
+     "rust_repositories", "rust_repository_set")
+rust_repositories(version = "1.47.0", edition="2018")
+
+load("@io_bazel_rules_rust//:workspace.bzl", "bazel_version")
+bazel_version(name = "bazel_version")
+
+################################################################
+################################################################
+# For local (independent) builds, this local repo allows us to import
+# marlin, which imports zexe. Some of marlin's targets depend on
+# @zexe// packages; this repo allows Bazel to resolve them here. In
+# addition, zexe targets that depend on both zexe and marlin packages
```

Comment:
This is no longer true now `snarky-bn382` is deprecated. Can we remove the dependency-cycle-break stuff for marlin/zexe here?

Reply:
done